lnst/Recipes/ENRT/SimpleMacsecRecipe.py

```diff
@@ -98,6 +93,7 @@ def generate_sub_configuration_description(self, config):
     def remove_sub_configuration(self, config):
         host1, host2 = config.host1, config.host2
         for host in (host1, host2):
+            config.untrack_device(host.msec0)
```

Comment:
Why is this required?

Reply:
After offline discussion, I understand why this is required. Simply to avoid corruption in multiple sub configuration runs.